src/Factories/Annotations/TestDox.php

```diff
@@ -14,13 +14,12 @@ final class TestDox implements AddsAnnotations
      */
     public function __invoke(TestCaseMethodFactory $method, array $annotations): array
     {
-        /*
-       * escapes docblock according to
+      /*
+       * Escapes docblock according to
        * https://manual.phpdoc.org/HTMLframesConverter/default/phpDocumentor/tutorial_phpDocumentor.howto.pkg.html#basics.desc
        *
-       * note: '@' escaping is not needed as it cannot be the first character of the line (it always starts with @testdox
+       * note: '@' escaping is not needed as it cannot be the first character of the line (it always starts with @testdox).
```

Comment:
Perhaps we can upper-case `Note` here too. 🤷🏻
```suggestion
       * Note: '@' escaping is not needed as it cannot be the first character of the line (it always starts with @testdox).
```